src/lint.js

```diff
@@ -3,6 +3,8 @@
 const CLIEngine = require('eslint').CLIEngine
 const path = require('path')
 const formatter = CLIEngine.getFormatter()
+const reallyRequire = require('really-require')
+require('colors')
```

Comment:
use `colors/safe` instead and use the exported value, instead of overloading `String` like this would do.

Reply:
In fact, we already have chalk so let's just use that. https://github.com/ipfs/aegir/blob/061d90a7870a624e194b89c81e742450e8903b5f/package.json#L44